instrumentation/jdbc/library/README.md

```diff
@@ -65,14 +65,14 @@ public class DataSourceConfig {
 
 #### Driver way
 
-1. Activate tracing for JDBC connections by setting `jdbc:otel:` prefix to the JDBC URL:
+* Activate tracing for JDBC connections by setting `jdbc:otel:` prefix to the JDBC URL:
```

Comment:
Can we go back to the ordered list here? I think these points are meant to be instructions that you should execute in order.

Reply:
OK, I fixed it to pass the markdown-lint check